orchestrate/local/my_airflow/requirements.txt

```diff
@@ -8,7 +8,7 @@ apache-airflow-providers-amazon==8.19.0
 apache-airflow-providers-celery==3.6.1
 apache-airflow-providers-cncf-kubernetes==9.0.0
 apache-airflow-providers-common-io==1.3.0
-apache-airflow-providers-common-sql==1.11.1
+apache-airflow-providers-common-sql>=1.24.1
```

Comment:
@BAntonellini can you check the Datacoves defaults for this, Github suggested we upgrade
cc/ @sconley-datacoves